src/components/AudioControls/AudioControls.js

```diff
@@ -0,0 +1,46 @@
+import { Pause, Play } from 'react-feather'
+
+import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
+
+AudioControls.propTypes = {
+  isPlaying: PropTypes.bool,
+  onPlayPauseClick: PropTypes.func,
+}
+
+export default function AudioControls({ isPlaying, onPlayPauseClick }) {
```

Comment:
her it is `isPlaying` which is probably something different.

Reply:
Thank you i changed it to a better name